tox.ini

```diff
@@ -1,74 +1,45 @@
 [tox]
+# generative list of environments (matrix) to run by default
 envlist =
-    clean,
-    py27-d1.7, py26-d1.8,
-    py33-d1.7, py33-d1.8,
-    py34-d1.7, py34-d1.8,
-    pypy-d1.7, pypy-d1.8,
-    pypy3-d1.7, pypy3-d1.8,
-    report
-[testenv]
-commands= coverage run -m unittest discover
-
-[testenv:clean]
-deps=
-  coverage
-commands=
-  coverage erase
-
-[testenv:report]
-deps=
-  coverage
-commands=
-  coverage report --include "cbs/*"
-  coverage html --include "cbs/*"
-
-# Python 2.7
-
-[testenv:py27-d1.7]
-deps = django>=1.7,<1.8
-  coverage
-
-[testenv:py27-d1.8]
-deps = django>=1.8,<1.9
-  coverage
+  clean,
+  py{27,33,34,35,pypy,pypy3}-django{1.6,1.7,1.8,1.9},
```

Comment:
Doesn't this mean that, along with py27, py33 and so on, we'll have pypypy and pypypy3 ?
I would expect it to be pypy and pypy3... does this fail silently with `skip_missing_interpreter`?